samples/app-ReporterPlus/ReporterPlus/Helpers/Constants.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace ReporterPlus.Helpers
+{
+    public static class Constants
+    {
+        public static string MicrosoftAppId;
+        public static string MicrosoftAppPassword;
+        public static string TenantId;
+        public static string ServiceUrl;
+        public static string BaseUrl;
+        public static string OriginatorId;
+        public static string SenderEmail;
+        public static string BlobContainerName;
+        public static string BlobConnectionString;
+        public static int MaxImagesCount = 5;
+        public static string MailSubject = "New Request Assigned";
+        public static string MemberGenericImageUrl = "https://devicecapabilities.blob.core.windows.net/filestorage/UserLogo.png";
```

Comment:
Why don't we add these images in this project in a resources folder and use them from the service? (We have the ngrok base url in the appsettings.json)

Reply:
We are using the other images that are created dynamically from the blob. As it would be difficult to use few from the local resource folder and few from the blob storage, made this generalized to use the blob url's all over.